core/src/main/java/tc/oc/pgm/spawner/objects/SpawnableItem.java

```diff
@@ -11,18 +11,18 @@
 
 public class SpawnableItem implements Spawnable {
 
-  private ItemStack stack;
-  private String METADATA_VALUE;
+  private final ItemStack stack;
+  private final String metadataValue;
```

Comment:
```suggestion
  private final String spawnerId;
```
Just rename it to something better